docs/content/docs/build/dds.md

```diff
@@ -185,23 +190,21 @@ Recall that the [SharedMap uses a last writer wins merge strategy](#merge-behavi
 editing the data at the same time, then the one who made the most recent change will overwrite the changes made by the
 other user.
 
-Imagine that you are collaborating with a colleague, and you change the shape's width while your colleague changes the
-shape's height. This will generate two operations: a `set` operation for you, and another `set` operation for your
-colleague. Both operations will be sequenced by the Fluid service, but only one will 'win,' because the SharedMap's
-merge behavior is LWW. Since we're storing the shape as an object, both `set` operations *set the whole object*.
+Imagine that a user "A" is collaborating with a colleague, and the user changes the shape's width while the colleague "B" changes the
+shape's height. This will generate two operations: a `set` operation for user A's change, and another `set` operation for user B's change. Both operations will be sequenced by the Fluid service, but only one will 'win,' because the SharedMap's merge behavior is LWW. Since we're storing the shape as an object, both `set` operations *set the whole object*.
```

Comment:
```suggestion
shape's height. This will generate two operations: a `set` operation for user A's change, and another `set` operation for user B's change. Both operations will be sequenced by the Fluid service, but only one will 'win,' because the SharedMap's merge behavior is LWW. Because the shape is stored as an object, both `set` operations *set the whole object*.
```